chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductCsvPresenter.java

```diff
@@ -107,6 +117,31 @@ public Logger getLogger() {
         result.add(data.getCertifyingBody().get("name").toString());
         result.add(data.getOtherAcb());
         result.add(data.getDeveloper().getName());
+        if (data.getDeveloper().getAddress().getLine2() != null) {
```

Comment:
Should this be "isEmpty"?

Also, for the Developer components, if they're empty (maybe no contact information, maybe something else) does the add that you have here correctly put a blank value in the csv? Or, if one of them is null, does it just not add the column at all?

Reply:
According to the definition files, those fields are required, which I think means that they will always have info?

Reply:
They might be required, but if the Developer doesn't have any contact information (like for Developers who only have 2011 listings, or only 2014 listings that haven't been updated, or have new ones) then there won't be data to show